Sources/DatadogObjc/DatadogConfiguration+objc.swift

```diff
@@ -331,6 +331,11 @@ public class DDConfigurationBuilder: NSObject {
         _ = sdkBuilder.set(serviceName: serviceName)
     }
 
+    @objc
+    public func set(loggingSamplingRate: Float) {
+        _ = sdkBuilder.set(loggingSamplingRate: loggingSamplingRate)
+    }
```

Comment:
We have some bad memories (https://github.com/DataDog/dd-sdk-ios/pull/583, https://github.com/DataDog/dd-sdk-ios/pull/359, https://github.com/DataDog/dd-sdk-ios/pull/360, https://github.com/DataDog/dd-sdk-ios/pull/461) with adding new APIs through `@objc` and not having them available in Objective-C. To anticipate problems, we added smoke tests in [DDConfiguration+apiTests.m](https://github.com/DataDog/dd-sdk-ios/blob/feature/session-replay/Tests/DatadogTests/DatadogObjc/ObjcAPITests/DDConfiguration%2BapiTests.m) - where each new Obj-c API can be called in `.m` file. Let's include this new `set(loggingSamplingRate` there, for 🧠 sanity 🙌.